source/Account.cpp

```diff
@@ -232,11 +245,6 @@ string Account::Step(int64_t assets, int64_t salaries, int64_t maintenance)
 	
 	out << "You paid ";
 	
-	auto creditString = [](int64_t payment) -> string
-	{
-		return payment == 1 ? "1 credit" : Format::Credits(payment) + " credits";
-	};
-	
 	map<string, int64_t> typesPaid;
 	if(salariesPaid)
 		typesPaid["crew salaries"] = salariesPaid;
```

Comment:
The line below should be `if(maintenancePaid > 0)`, but GitHub doesn't allow me to write a comment on that line.